cmd/copy.go

```diff
@@ -221,6 +227,10 @@ func (raw rawCopyCmdArgs) stripTrailingWildcardOnRemoteSource(location common.Lo
 
 func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, error) {
 
+	// White Glove - Set Webhook URI and Project ID UUID from the parameters passed in the command line
+	telemetry.ReportingAPIURI = raw.reportingAPI
+	telemetry.ProjectReferenceID = raw.projectID
```

Comment:
Consider using environment variables to set these, or moving them to the root command.